### DISCUSSION TITLE
IPv6-Only Mode

### DISCUSSION THREAD

**Discussion by ghost** [mullvad/mullvadvpn-app#9122] [Feature requests & Ideas]
### Issue description
Currently Mullvad offers IPv4-only mode and dual stack mode.
Could you please add IPv6-only mode?
It would be useful to have for debugging websites.

**Comment by faern** [comment]
That would be a bit messy. We currently rely on a bunch of v4-only endpoints for the app to function. But we can look at how hard it would be to make those toggle over to some other addrs when v6 is available

**Comment by L-as** [comment]
This would be immensely useful as a solution to #3711.

**Comment by alexhaydock** [comment]
> We currently rely on a bunch of v4-only endpoints for the app to function.

Are there any plans to amend this? Would be useful for v6-only networks where CLAT/464XLAT isn't available.

**Comment by faern** [comment]
I have raised the issue internally and created tickets for the services that are currently IPv4 only. But there is no timeline or anything for when this might be fixed.
